cibyl/sources/elasticsearch/api.py

```diff
@@ -81,9 +81,52 @@ def __query_get_hits(self: object, query: dict, index: str = '') -> list:
         try:
             response = self.es_client.search(
                 index=index,
-                body=query
+                body=query,
+                size=1000
             )
         except Exception as exception:
             raise ElasticSearchError("Error getting the results") \
                   from exception
         return response['hits']['hits']
+
+    def get_builds(self, **kwargs):
+        """
+            Get builds from elasticsearch server.
+
+            :returns: container of jobs with build information from
+            elasticsearch server
+            :rtype: :class:`AttributeDictValue`
+        """
+        jobs_found = self.get_jobs(**kwargs)
+
+        for job_name, job in jobs_found.items():
+            query_body = QueryTemplate('job_name', job_name).get
+
+            builds = self.__query_get_hits(
+                index='jenkins_builds',
```

Comment:
can this be index=self.index? so users can modify it from configuration

Reply:
It depends how we handle it. 

The thing is we’re using in one method one index (`get_jobs` with Jenkins) and in the other method another different index (`get_builds` with jenkins_builds).

Reply:
Yeah so probably no. But I wish we didn't have to include indexes names hardcoded in the code

Reply:
Maybe we can pass in the configuration file `jobs_index` and `builds_index` and we’ll be able to receive it as you commented. 

I’ll also think another options.

Reply:
maybe. but will we need also "tests_index"?
why do we actually have two indexes for builds and jobs? can't we just use one?

Reply:
Jobs and builds are in different indexes. The relationship with them is the job name. 
From one side I can get all the jobs from one index. For being able to get the builds I search them by the job name. That's why I need to specify the index for every search.

Reply:
Looking at "builds_index" I see for every document the fields: "job_name", "build_result", ... why can't use only "builds_index"?

Reply:
k, we can do index optimization later on. Let's merge it when CI passes

Reply:
Ok, I'll finish it.